README.md

```diff
@@ -600,3 +602,17 @@ Future plans involve adding support for the following features:
 
 - 📄 Run addon reports
 - ⚙️ Spawning Storybook via the test runner in a single command
+
+---
+
+## Contributing
+
+We welcome contributions to the test runner!
+
+### Branch/release structure
+
```

Comment:
Do you think it makes sense adding a small recap somewhere here?

In short:
- `main` - refers to `latest` version on npm.
  - Used for automated releases of the stable version of this package.
- `main-prerelease` - refers to `prerelease` version on npm. 
  - Used for cherry-picking fixes from `next` and creating fixes that don't make sense to bring to `next`.
- `next` - refers to `next` version on npm. 
  - Default branch, used for most of the work. Only branch off of another branch if really necessary.